usecase/tenantd.go

```diff
@@ -138,15 +138,23 @@ func (t *clientd) Start(ctx context.Context) chan []error {
 	if t.access != nil {
 		go func() {
 			for err := range t.access.StartAccessUpdater(ctx) {
-				glg.Error(err)
+				if err == ctx.Err() {
+					glg.Info("Stopped access token updater")
+				} else {
+					glg.Errorf("StartAccessUpdater error: %s", err.Error())
+				}
```

Comment:
```suggestion
if err == ctx.Err() {
	glg.Info("Stopped access token updater")
	continue
}

glg.Errorf("StartAccessUpdater error: %s", err.Error())
```

まあ、Elseは避けてもいいかなと思いました。